src/Resources/views/CRUD/base_array_macro.html.twig

```diff
@@ -8,17 +8,42 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% macro render_array(value, inline) %}
-    {% from _self import render_array %}
-    {% for key, val in value %}
-        {% if val is iterable %}
-            [{{ key }} => {{ render_array(val, inline) }}]
-        {% else %}
-            [{{ key }} => {{ val }}]
-        {% endif %}
+{% macro render_array(value, inline, values_only, translate, translation_domain, translation_prefix, translate_key) %}
+    {% spaceless %}
+        {% if not inline %}<ul>{% endif %}
+        {% from _self import render_array %}
+        {% for key, val in value %}
+            {% if inline == true %}
```

Comment:
@osavchenko Can you add a screenshot with the inline option set to true to show no regression ?

Reply:
![image](https://user-images.githubusercontent.com/617002/78592314-97a1ce00-784d-11ea-8401-d5ede99c0a66.png)
Only hew extra spaces around

Reply:
Added screenshot to the PR description with a new inline view.